helios-services/pom.xml

```diff
@@ -319,6 +318,13 @@
       <version>0.8</version>
     </dependency>
 
+    <!-- versions that we have to manage to avoid requireUpperBoundDeps violations -->
```

Comment:
ah I see httpcore is in dependencyManagement in the parent pom, so why not manage both there?